tests/integration/test_case.go

```diff
@@ -251,3 +251,14 @@ type IntrospectionRequest struct {
 	// contains this string.
 	ExpectedError string
 }
+
+type ClientIntrospectionRequest struct {
```

Comment:
todo: Please provide some type-level documentation for this :)

Reply:
resolved